audio/pulseaudio/Portfile

```diff
@@ -31,7 +31,7 @@ checksums           rmd160  5e52b8549a0ece2b52df854b5edaa2998c438ccd \
                     sha256  75d3f7742c1ae449049a4c88900e454b8b350ecaa8c544f3488a2562a9ff66f1 \
                     size    1951300
 
-set perl_branch     5.28
+set perl_branch     5.34
```

Comment:
This is only used by `depends_build`. Maybe it could switch to the `perl5` portgroup and simply use whatever is defined as the default perl there.